lib/Cake/Network/CakeResponse.php

```diff
@@ -68,6 +68,7 @@ class CakeResponse {
 		415 => 'Unsupported Media Type',
 		416 => 'Requested range not satisfiable',
 		417 => 'Expectation Failed',
+		429 => 'Too Many Requests',
```

Comment:
Yeah, I had a newb moment and didn't cherry-pick my commits. I can re-do the pull request if it would make things easier.